src/utils/renderProps.js

```diff
@@ -10,10 +10,12 @@ const isFn = (prop) => typeof prop === 'function'
  */
 
 const renderProps = ({ children, render }, props) => {
-  warn(isFn(children) && isFn(render), `
-    You are using the children and render props together.
-    This is impossible, therefore, only the children will be used.
-  `)
+  if (process.env.NODE_ENV !== 'production') {
+    warn(isFn(children) && isFn(render),
+      'You are using the children and render props together.' +
```

Comment:
Put space after dot please

Reply:
Or newline

Reply:
oh, mi intention was a new line, forgot to add this. Good catch 👍